furms-core/src/main/java/io/imunity/furms/core/project_installation/ProjectInstallationMessageResolverImpl.java

```diff
@@ -46,6 +46,9 @@ public void update(CorrelationId correlationId, ProjectInstallationResult result
 			projectAllocationInstallationService.startWaitingAllocations(job.projectId);
```

Comment:
Comment for line 40: this is an error prone practice. Instead in the ProjectInstallationStatus add a method isTerminal() and use it here. Same for the other cases like this.